csharp/Platform.IO/StreamExtensions.cs

```diff
@@ -25,6 +58,22 @@ public static T ReadOrDefault<T>(this Stream stream)
             return stream.Read(buffer, 0, size) == size ? buffer.ToStructure<T>() : default;
         }
 
+        /// <summary>
+        /// <para>Reads and returns array of <typeparam name="T"> structures from the <paramref name="stream"/>.</para>
+        /// <para>Прочитывает и возвращает массив всех структур <typeparam name="T"> из потока <paramref name="stream"/>.</para>
+        /// </summary>
+        /// <typeparam name="T">
+        /// <para>The structure type.</para>
+        /// <para>Тип являющийся структурой.</para>
+        /// </typeparam>
+        /// <param name="stream">
+        /// <para>A stream containing the <typeparam name="T"> structure.</para>
```

Comment:
`the <typeparam name="T"> structure` ⇾ `the <typeparam name="T"> structure value`